external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaContinuousSourceSuite.scala

```diff
@@ -169,6 +169,54 @@ class KafkaContinuousSourceSuite extends KafkaSourceSuiteBase with KafkaContinuo
       }
     }
   }
+
+  test("read kafka record containing null key/values") {
```

Comment:
Thank you for updating. For a bug test case, we add JIRA issue like the following.
```scala
- test("read kafka record containing null key/values") {
+ test("SPARK-27494 read kafka record containing null key/values") {
```

Reply:
ok